roles/rsyslog/vars/main.yml

```diff
@@ -51,3 +76,12 @@ rsyslog_weight_map:
   'rulesets': '50'
   'input': '90'
   'inputs': '90'
+
+# X.509 certificates related variables
+# Lookup rsyslog netwrok stream driver
+__rsyslog_pki_map:
+  none: ptcp
+  ptcp: ptcp
+  tls: gtls
+  gtls: gtls
+  gnutls: gtls
```

Comment:
Was there an `openssl` in this list, or was that another list?

Reply:
First I put openssl, but I noticed we don't enable openssl in building rsyslog package for rhel's. So, I removed it (but I forgot to clean up all of them... Sorry, I'm going to do it. ):
```
# cat /etc/redhat-release 
CentOS Linux release 8.1.1911 (Core) 
# yum search rsyslog | egrep ssl
# yum search rsyslog | egrep gnu
rsyslog-gnutls.x86_64 : TLS protocol support for rsyslog
```

Reply:
> First I put openssl, but I noticed we don't enable openssl in building rsyslog package for rhel's

What about Fedora?  Is there openssl support there?

Reply:
> > First I put openssl, but I noticed we don't enable openssl in building rsyslog package for rhel's
> 
> What about Fedora? Is there openssl support there?

No, it's not supported in Fedora 31 and 32...